app/backend/src/couchers/jobs/handlers.py

```diff
@@ -6,13 +6,15 @@
 from datetime import timedelta
 
 import requests
+from sqlalchemy import delete
 from sqlalchemy.sql import func, or_
 
 from couchers import config, email, urls
 from couchers.db import session_scope
 from couchers.email.dev import print_dev_email
 from couchers.email.smtp import send_smtp_email
 from couchers.models import GroupChat, GroupChatSubscription, HostRequest, LoginToken, Message, MessageType, User
+from couchers.sql import couchers_select as select
```

Comment:
I think being explicit using couchers_select everywhere makes it easier to read the  code without having to scroll up.

Reply:
I chose to name it `couchers_select` instead of `select` so that it's clear to a future developer we are using our own construct, and so that they don't accidentally import normal `select` and wonder why it's not working

I chose to rename it `select` so that our sql statements are as similar to normal SQLA syntax as possible, 1) for clarity and 2) for troubleshooting

Essentially I wanted it to look like normal SQLA, but I wanted to leave easy breadcrumbs so that some dev in 5 years can pick it up and figure it out

Reply:
OK, if you both think it's better, let's do this!

Reply:
I think @kalvdans means to just import couchers_select and do session.execute(couchers_select(User))

So there's three options

Reply:
We never really resolved this. But we can discuss and if we want to rename, it's no big deal

Reply:
Could you remind us to discuss it in the next backend meeting?